### PR TITLE
update track and track composite

### PR DESCRIPTION
**Pull request by frostbyte73**

### CODE REVIEW

File: livekit_egress.proto [hunk 1]
```diff
@@ -60,16 +60,20 @@ message TrackCompositeEgressRequest {
 message TrackEgressRequest {
   string room_name = 1;  // required
   string track_id = 2;   // required
+  bool containerize = 3; // does not work with websocket output
```
Comment (davidzhao) [inline livekit_egress.proto:63]: what does this do?

File: livekit_egress.proto [hunk 2]
```diff
@@ -60,16 +60,20 @@ message TrackCompositeEgressRequest {
...
+    AzureBlobUpload azure = 6;
+    string websocket_url = 7;
   }
 }
 
 enum EncodedFileType {
-  MP4  = 0;
-  OGG = 1;
-//  WEBM = 2; coming soon
+  DEFAULT = 0;
```
Comment (davidzhao) [inline livekit_egress.proto:73]: some comments on what this is would be great!

Comment (davidzhao) [reply livekit_egress.proto:73]: and EncodedFileOutput.file_type would become optional then?

File: livekit_egress.proto (hunk 1 above)
Comment (frostbyte73) [reply livekit_egress.proto]: I was confused, if you want containerization you'd use track composite

File: livekit_egress.proto (hunk 2 above)
Comment (frostbyte73) [reply livekit_egress.proto]: yeah, and we'd choose it based on the input codecs and available options. Added comments for both

**Review by davidzhao — APPROVED** [review]

**Pull request merged**